src/algebra/lie/of_associative.lean

```diff
@@ -62,6 +62,8 @@ instance of_associative_ring : lie_ring A :=
 
 lemma of_associative_ring_bracket (x y : A) : ⁅x, y⁆ = x*y - y*x := rfl
 
+lemma lie_apply {α : Type*} (f g : α → A) (a : α) : ⁅f, g⁆ a = ⁅f a, g a⁆ := rfl
```

Comment:
```suggestion
section pi

variables {α : Type*} (f g : α → A)

@[simp] lemma coe_lie : (⁅f, g⁆ : α → A) = λ a, ⁅f a, g a⁆ := rfl

lemma lie_apply (a : α) : ⁅f, g⁆ a = ⁅f a, g a⁆ := rfl

end pi
```

Reply:
Introducing the binders in a simp lemma like coe_lie is an unusual choice; we don't do it for addition.

Also, there's no actual coercion happening there!

Reply:
Oops, the `coe_` prefix was a bit of a thinko: I've seen so many of these `_apply` lemmas being written for bundled morphisms and I like to set things up like this in that case. Let's drop this idea here since it's just a distraction from an already-large PR.